ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Workbenches.resource

```diff
@@ -122,19 +122,13 @@ Add Environment Variables In Workbench
                 Log   ${pair_idx}-${key}-${value}
                 Run Keyword And Continue On Failure    Wait Until Page Contains Element
                 ...    xpath=//input[@aria-label="key of item ${pair_idx}"]
-                Input Text    ${ENV_VARIABLES_SECTION_XP}/div[@class="pf-l-split"][${idx}]//input[@aria-label="key of item ${pair_idx}"]
+                Input Text    ${ENV_VARIABLES_SECTION_XP}/div[contains(@class, "-l-split")][${idx}]//input[@aria-label="key of item ${pair_idx}"]
                 ...           ${key}
-                Input Text    ${ENV_VARIABLES_SECTION_XP}/div[@class="pf-l-split"][${idx}]//input[@aria-label="value of item ${pair_idx}"]
+                Input Text    ${ENV_VARIABLES_SECTION_XP}/div[contains(@class, "-l-split")][${idx}]//input[@aria-label="value of item ${pair_idx}"]
                 ...           ${value}
                 IF    ${n_pairs}-${pair_idx} > 1
-                    Click Element    ${ENV_VARIABLES_SECTION_XP}/div[@class="pf-l-split"][${idx}]//button[text()="${add_key_value_text}"]
-                    # The element we want to click might get pushed out of view on every loop, let's scroll by an approx
-                    # amount of pixels for the block of elements that gets added.
-                    IF    "${add_key_value_text}" == "Add another key / value pair"
-                        Execute Javascript    document.getElementsByClassName("pf-c-drawer__content")[1].scrollBy(0,500)  # robocop: disable
-                    ELSE IF    "${add_key_value_text}" == "Add another variable"
-                        Execute Javascript    document.getElementsByClassName("pf-c-drawer__content")[1].scrollBy(0,700)  # robocop: disable
-                    END
```

Comment:
I think this might be still needed in case of many env variables, try running ODS-2595 for example

Reply:
I'm not just deleting it, I'm navigating using the left bar menu, instead of usin js scripts

Reply:
I see that, but it might not be enough when the variables are many. Pls try with ODS-2595

Reply:
Added "do not merge" label until everything related with this is properly tested and or fixed

Reply:
Just using the navigation when creating multiple env vars does not work properly.
I've restored the script and modified it in order to make it work with the new UI

Reply:
@bdattoma Can you please check the last changes? If you are OK with the new script I will merge this PR

Reply:
:facepalm:  my bad....
@bdattoma can you re review?